app/views/homepage/popular_links/_sidebar.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar__button sidebar__border">
+  <h2 class="sidebar__heading">Options</h2>
+  <% if @latest_popular_links.state == "published" %>
+    <%= primary_theme_navigation_button("Create new edition", :post, create_popular_links_path) %>
+  <% else %>
+    <%= primary_theme_navigation_button("Edit popular links", :get, edit_popular_links_path(@latest_popular_links)) %>
```

Comment:
Do these buttons need to be part of a form or can we use the `Link button` variation of the [Button component](https://components.publishing.service.gov.uk/component-guide/button) and supply the URL for the edit page?

Reply:
we did look at this and tried this but from what I understand, it wont be possible to use button components as it would not allow me to do a POST request.

Reply:
I think these are simple links really to the edit page so we should mark them up as that and wouldn't need to do a POST request via a form. Unless I have missed something.

Reply:
I'm still not sure why these are not rendered as links: I can't see the advantage of using a button in a form for this. I think the precedent in Whitehall would be to use a link, seems to be the equivalent of "Edit draft" on [this page](https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/detailed-guides/1575767).

Reply:
I agree that the edit "button" could be a link, since it's a get request. The "Create new edition" is actually causing a data change on the backend (creating a new edition), so should be a POST request (and so needs a form with submit button).

Reply:
(for the create button that would match [what Whitehall does](https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/publications/1571182)).

Reply:
We have made the button for the 'GET' request to edit page to be a `Link button` now.